YouVsVirus/Assets/Scripts/Components/EndLevelControllerBase.cs

```diff
@@ -152,7 +156,80 @@ public virtual void NotifyInt (int data)
+    {
+
+        //  Removes this human's Rigidbody from the physics simulation, 
+        //  which disables movement and collision detection.
+        human.GetComponent<Rigidbody2D>().simulated = false;
+        //  Set the simulated velocity to zero.
+        human.GetComponent<Rigidbody2D>().velocity = Vector3.zero;
+        // Set position of player hardcoded, see below for how it might
+        // work automatically
+        human.transform.position = position;
+        Debug.Log(human.transform.localScale);
+        // make player twice as large
+        human.transform.localScale *= 2;
+        // make player and every sprite attached to player appear on top of everything
+         SpriteRenderer[] renderers = human.transform.GetComponentsInChildren<SpriteRenderer>();
+        foreach (SpriteRenderer sr in renderers)
+        {
+            sr.sortingLayerName = "OnTop"; 
+        }
+    }
+
+    /// <summary>
+    /// Find the player and their friend if possible and assign them a position
+    /// </summary>
+    /// <param name="canv"> the canvas whose corner we want to place them on </param>
+    protected void FindAndPlaceHuman(GameObject canv)
+    {
+        // Get a reference to the player if possible.
+        if(GameObject.FindWithTag("Player") != null)
+        {
+            Components.Player player = GameObject.FindWithTag("Player").GetComponent<Components.Player>();
+            // Set position of player hardcoded, see below for how it might
+            // work automatically
+            Vector3 position = new Vector3(3.53f, -2.53f, 0.0f);
+            PlaceAndStopHuman(position, player);
+        }
+        else Debug.Log("No game object with name 'Player' found");
+
+        // Get a reference to the friend if possible.       
+        if (GameObject.FindWithTag("Friend") != null)
+        {
+            Components.Friend friend = GameObject.FindWithTag("Friend").GetComponent<Components.Friend>();
+            Debug.Log("friend");
+            // Set position of friend hardcoded, see below for how it might
+            // work automatically
+            Vector3 position = new Vector3(-2.79f, -2.53f, 0.0f);
+            PlaceAndStopHuman(position, friend);
+        }
+        else Debug.Log("No game object with name 'Friend' found");
+
+
+        // This is how we could get the corners of the panel
+        // however the corners are not completely right if we use those
+        //Vector3[] corners = new Vector3[4];
+        //GameObject Panel = canv.transform.Find("Panel").gameObject;
+        ////If the child was found.
+        //if (Panel != null)
+        //{
+        //    Panel.GetComponent<RectTransform>().GetWorldCorners(corners);
+        //    player.transform.position = new Vector3(Mathf.Abs(corners[0].x), -Mathf.Abs(corners[0].y), corners[0].z);
+        //    player.transform.localScale *= 2;
+        //}
+        //else Debug.Log("No child with the name 'Panel' attached to the canvas");
```

Comment:
Maybe `Panel.GetComponent<Image>().GetWorldCorners` could work?

Reply:
I am pretty sure I tried this first and it gives exactly the same result as   
Panel.GetComponent<RectTransform>().GetWorldCorners(corners);